lighthouse-core/gather/gather-runner.js

```diff
@@ -115,6 +115,24 @@ class GatherRunner {
     return {};
   }
 
+  /**
+   * Reject if the gathering terminates due to the CDP target crashing, etc
```

Comment:
non-review comment below

Worth noting here for lurkers that this is a good example our long-standing tradition of memory leak listeners that are never removed and promises that never die. We have a ton of these, so shouldn't need to do anything differently here, but a few folks have asked recently about why we recommend child processes and disposable contexts, this is why :)